src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -920,25 +920,34 @@ private static RestoreLockProperties GetRestoreLockProperties(IMSBuildItem specI
                 IsPropertyTrue(specItem, "RestoreLockedMode"));
         }
 
-        public static RestoreAuditProperties GetRestoreAuditProperties(IMSBuildItem specItem)
+        public static RestoreAuditProperties GetRestoreAuditProperties(IMSBuildItem specItem, IList<string> suppressionItems)
         {
             string enableAudit = specItem.GetProperty("NuGetAudit");
             string auditLevel = specItem.GetProperty("NuGetAuditLevel");
             string auditMode = specItem.GetProperty("NuGetAuditMode");
 
-            if (enableAudit != null || auditLevel != null || auditMode != null)
+            if (enableAudit != null || auditLevel != null || auditMode != null
+                || (suppressionItems != null && suppressionItems.Count() > 0))
```

Comment:
```suggestion
                || (suppressionItems != null && suppressionItems.Count > 0))
```

Never call LINQ methods when not needed. They cause memory allocations, plus require a bunch of CPU work to calculate the value that already exists.

Reply:
Fixed.